src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -74,49 +74,49 @@ class TinyMCEConfig extends HTMLEditorConfig
         'fa_PK' => 'fa',
         'fi_FI' => 'fi',
         'fi_SE' => 'fi',
-        'fr_BE' => 'fr',
-        'fr_BF' => 'fr',
-        'fr_BI' => 'fr',
-        'fr_BJ' => 'fr',
-        'fr_CA' => 'fr_ca',
-        'fr_CF' => 'fr',
-        'fr_CG' => 'fr',
-        'fr_CH' => 'fr',
-        'fr_CI' => 'fr',
-        'fr_CM' => 'fr',
-        'fr_DJ' => 'fr',
-        'fr_DZ' => 'fr',
-        'fr_FR' => 'fr',
-        'fr_GA' => 'fr',
-        'fr_GF' => 'fr',
-        'fr_GN' => 'fr',
-        'fr_GP' => 'fr',
-        'fr_HT' => 'fr',
-        'fr_KM' => 'fr',
-        'fr_LU' => 'fr',
-        'fr_MA' => 'fr',
-        'fr_MC' => 'fr',
-        'fr_MG' => 'fr',
-        'fr_ML' => 'fr',
-        'fr_MQ' => 'fr',
-        'fr_MU' => 'fr',
-        'fr_NC' => 'fr',
-        'fr_NE' => 'fr',
-        'fr_PF' => 'fr',
-        'fr_PM' => 'fr',
-        'fr_RE' => 'fr',
-        'fr_RW' => 'fr',
-        'fr_SC' => 'fr',
-        'fr_SN' => 'fr',
-        'fr_SY' => 'fr',
-        'fr_TD' => 'fr',
-        'fr_TG' => 'fr',
-        'fr_TN' => 'fr',
-        'fr_VU' => 'fr',
-        'fr_WF' => 'fr',
-        'fr_YT' => 'fr',
-        'fr_GB' => 'fr',
-        'fr_US' => 'fr',
+        'fr_BE' => 'fr_FR',
+        'fr_BF' => 'fr_FR',
+        'fr_BI' => 'fr_FR',
+        'fr_BJ' => 'fr_FR',
+        'fr_CA' => 'fr_FR',
```

Comment:
Does this not need to be `fr_ca` as currently or does that not work?

Reply:
I didn't test, but there is no reason why it should work because there is only one js file which is fr_FR.js

Reply:
ok